pkg/client/client.go

```diff
@@ -73,6 +73,17 @@ func (c *Client) QueueStatus(routingKey string) (*http.Response, error) {
 	return c.Do(req)
 }
 
+func (c *Client) HealthCheck() (*http.Response, error) {
+	url := generateURL(c.ServerAddress, "/health")
+
+	req, err := http.NewRequest("GET", url.String(), nil)
+
+	if err != nil {
+		return nil, err
+	}
+	return c.Do(req)
```

Comment:
The body from the `*http.Response` here needs to be closed at some point, either here (which would take some refactoring) or in `runHealthCommand`.

Reply:
ah good catch.  It actually doesn't look like we close the response body for any of the CLI commands.  My guess is maybe we have ignored this since the CLI commands are short-lived and the file descriptors get closed on exit?

Reply:
True, it's fairly low risk. 

Giving a 👍, can always do a pass to cinch things up later.